documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -247,364 +239,404 @@ Update this tool to the latest version.
 [[about]]
 ==== About raspi-config
 
-Selecting this option shows the following text:
-
-----
-This tool provides a straightforward way of doing initial configuration of the Raspberry Pi.
-Although it can be run at any time, some of the options may have difficulties if you have heavily customised your installation.
-----
+Selecting this option displays a description of `raspi-config`.
 
 [[finish]]
 ==== Finish
 
-Use this option when you have completed your changes. You will be asked whether you want to reboot or not. When implementing changes for the first time, it's best to reboot. There will be a delay in rebooting if you have chosen to resize your SD card.
+Use this option when you have completed your changes. You will be asked whether you want to reboot or not. When implementing changes for the first time, it's best to reboot. If you chose to resize your SD card, rebooting may take longer than usual.
 
 [[raspi-config-cli]]
-== The `raspi-config` Command Line Interface
+== The `raspi-config` CLI
 
-The `raspi-config` tool can also be run in a non-interactive mode, which is useful for setting up a Raspberry Pi image for distribution.
+The `raspi-config` tool supports non-interactive command line options and flags instead of the interactive menu. These options can be useful when configuring a Raspberry Pi image for distribution.
 
+[source,console]
 ----
-sudo raspi-config nonint <command> <arguments>
+$ sudo raspi-config nonint <command> <arguments> [optional-argument]
 ----
 
-The `sudo` is required because you will be changing files that are not owned by the default user.
-
-NOTE: There is no consistent meaning for `0` and `1` in arguments. Each function will document what `0` and `1` mean for that particular function.
+NOTE: The meaning of `0` and `1` varies between options. Always check the documentation before passing a value to an option.
 
 [[raspi-config-cli-commands]]
 === List of options
 
-NOTE: The `raspi-config` tool is under continuous development, and the list of options below may not be completely up to date. Please be aware that different models of Raspberry Pi may have different options available.
```

Comment:
Might be worth retaining this warning? Or it least adding the "Please be aware that different models of Raspberry Pi may have different options available." to the previous NOTE?

Reply:
Continuous development is so common at this point that it's no longer a great excuse for outdated doc -- if we're missing something, I hope someone opens an issue and calls it out!

As for the model differences, that's worth calling out in both the int and non-int sections. But probably not in an admonition :)